.travis.yml

```diff
@@ -0,0 +1,37 @@
+branches:
+  only: 
+    - master
+
+sudo: required
+services:
+  - docker
+
+before_install:
+  - docker build -t ekarychkovsky/team-balance-app-server-test -f ./server/Dockerfile.test ./server
+
+script:
+  - docker run ekarychkovsky/team-balance-app-server-test
+#    - docker build -f ./client/Dockerfile.ci ./client
+
+after_success:
+  - docker build -t ekarychkovsky/team-balance-app-client ./client
```

Comment:
why ekarychkovksy?

Reply:
Sorry, Haven't got time to create DockerHub account for our organization. So used my personal account for it.
Will create ticket for it.